migrate/migrator.go

```diff
@@ -227,12 +228,19 @@ func WithPackageName(name string) GoMigrationOption {
 	}
 }
 
+func WithContent(content string) GoMigrationOption {
```

Comment:
Please rename to `WithGoTemplate` so we can also have `WithSQLTemplate` later.

Reply:
Done.